theme/less/components/buttons.less

```diff
@@ -132,87 +98,29 @@ There are `.fr-btn--icon-left` and `fr-btn--icon-right` to add spacing around th
     }
 }
 
-
-.btn-primary {
-    .btn;
```

Comment:
`font-size` is 1rem now, instead of `@font-size-sm` with previous buttons (see `.btn` class). Can we get back to previous font size?